src/test/java/StringTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringTest {
+    @Test
+    void 문자열을_쪼깬_배열은_쪼개진_요소들은_포함한다() {
```

Comment:
```suggestion
    void 문자열을_쪼갠_배열은_쪼개진_요소들을_포함한다() {
```

사소해 보일 수 있지만, 한국어는 조사 하나로 의미가 달라지는 경우가 많아 때로는 작은 실수가 큰 비용을 유발하기도 하는 것 같아요. 꼼꼼히 확인하는 습관을 들여 보는 건 어떨까요? 😸

Reply:
더 신경을 쓰도록 하겠습니당